scripts/error-extractor/parseJSFile.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+var babylon = require('babylon');
+var fs = require('fs');
+
+function parseJSFile(filePath/* : string */)/* : Object */ {
```

Comment:
How about inlining this module and lifting the babylon config to a variable at the top?